totpauth-impl/src/main/resources/conf/authn/totp-authn-config.xml

```diff
@@ -11,6 +11,10 @@
 	<!-- Names of form fields to pull tokenNumber -->
 	<bean id="shibboleth.authn.tokenCodeField" class="java.lang.String" c:_0="j_tokenNumber" />
 
+	<!-- TOTP Seed DB related fields -->
+	<bean id="shibboleth.authn.seedDBTableName" class="java.lang.String" c:_0="dbtablename"/>
+	<bean id="shibboleth.authn.usernameColumnName" class="java.lang.String" c:_0="email"/>
+	<bean id="shibboleth.authn.seedColumnName" class="java.lang.String" c:_0="totpseed"/>
```

Comment:
These database columns will need to match what's in 18F/cg-provision#118.

https://github.com/LinuxBozo/cg-provision/blob/d7b6a8989209774f0b3167815b5d8555b7eb59a6/ci/scripts/create-and-update-db.sh#L26